src/app/components/Breadcrumbs/Breadcrumbs.jsx

```diff
@@ -51,12 +51,16 @@ const Breadcrumbs = ({ query, type, bibUrl, itemUrl, edd }) => {
       return crumbs;
     }
 
-    crumbs.push(
+    const stringifiedQuery = query.replace("q=", "")
```

Comment:
Hopefully `q=` doesn't occur anywhere in the query but I would still make sure we are only replacing `q=` at the beginning i.e. `query.replace(/^q=/, '')`